BuildLoader.py

```diff
@@ -1020,6 +1021,18 @@ def pre_build(self):
         if plt_dir != sbl_dir:
             os.environ['PACKAGES_PATH'] += os.pathsep + sbl_dir
 
+        # Generate SblKeys
+        if self._board.KEY_GEN and not os.path.exists(os.environ.get('SBL_KEY_DIR')):
+            print ("Generating default Sbl Keys..")

```

Comment:
Print the key path as well. 
print ("Generating default Sbl Keys in directory %s .." %  sbl_key_dir)